### PR TITLE
Unittest mock

### PR DESCRIPTION
**Pull request by Rwatana**
I forgot to make the Pull request. Please check.

### CODE REVIEW

File: get-resource/src/main.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package main
```
Comment (nayuta723) [inline get-resource/src/main.go]: Remove this file.

File: test/src/resource_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import (
+	"context"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fake "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestCreatePod(t *testing.T) {
```
Comment (nayuta723) [inline test/src/resource_test.go]: Test function must test a function.

Comment (nayuta723) [reply test/src/resource_test.go]: Please divide by two so as to test part and function part.

**Review by nayuta723 — CHANGES_REQUESTED** [review]

**Comment by Rwatana** [comment]
@nayuta-ai 
I removed the file and divide them please check.

File: get-resource/src/main.go (hunk 1 above)
Comment (nayuta723) [reply get-resource/src/main.go]: One More, Remove this file because this file covers PR1.

File: test/src/resource_test.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
+package main
```
Comment (nayuta723) [inline test/src/resource_test.go]: Change the directory structure, according to [Go言語でテストコードを書いてみよう](https://note.com/rescuenow_hr/n/n9ed7caf4646d)

File: test/src/resource_test.go (hunk 2 above)
Comment (nayuta723) [inline test/src/resource_test.go]: What function does this file test?

**Review by nayuta723 — CHANGES_REQUESTED** [review]

File: test/resource_test.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
+package main
```
Comment (nayuta723) [inline test/resource_test.go]: What purpose is this test file?

The purpose of unit test is to check whether target function's behavior is correct or not.

So, I reccomed to separate this file for target function and the corresponding test functioe.

Comment (nayuta723) [reply test/resource_test.go]:
```
|- mock-practice -- create_pod.go
                  |- create_pod_test.go
```

**Review by nayuta723 — CHANGES_REQUESTED** [review]

**Comment by Rwatana** [comment]
@nayuta-ai 
I think I have revised the code and the composition of them. Please check my code

**Comment by nayuta723** [comment]
Sorry, Please change as the below.
```
|- mock-practice -- create_pod.go
                  |- create_pod_test.go
```

Also, please delete `test` directory.

**Comment by Rwatana** [comment]
@nayuta-ai  Sure, I revised them.
Please check

**Pull request merged**